apk-generator/scripts/cleanup.sh

```diff
@@ -0,0 +1,2 @@
+find /var/www/html/release* -mtime +3 -exec rm -r {} \;
```

Comment:
Also go through the respective folder in /var/www/files/* and /var/www/html/uploads?